pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -458,6 +458,27 @@ struct LogicalOp {
   }
 };
 
+struct LogicalNotOp {
+  Value create(ConversionPatternRewriter &rewriter, Location loc,
+               Type resultType, ArrayRef<Value> operands,
+               ArrayRef<Type> types) {
+    auto &input = operands[0];
```

Comment:
`Value` types should always be treated with value-like semantics (since it's implemented as a smart-pointer). No need to take a reference here.